screens/LoginScreen.js

```diff
@@ -14,6 +14,15 @@ import {
 } from 'native-base';
 
 export default class LoginScreen extends React.Component {
+    submitForm = () => {
+        // Validate form has proper input
+
+        // Submit credentials
+
+        // In promise response, if validated, direct to main page
+        this.props.navigation.navigate('Main');
```

Comment:
bypassing proper auth for now so we can see the other screens too if you "login" :)